control-plane/pkg/reconciler/kafka/testing/MockKafkaClient.go

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
File naming is not consistent with others here

Reply:
True

/hold

Reply:
Done
/unhold